packager/react-packager/src/Resolver/polyfills/prelude.js

```diff
@@ -1,5 +1,4 @@
-/* eslint global-strict:0 */
-__DEV__ = false;
+/* eslint strict:0 */
+global.__DEV__ = false;
```

Comment:
We have a special transform which picks up this variable – I will check whether it still works